crates/libcruby-sys/src/lib.rs

```diff
@@ -3,6 +3,26 @@
 
 extern crate libc;
 
+use std::ffi::CString;
+
+pub const RUNTIME_VERSION: &'static str = env!("CARGO_PKG_VERSION");
+
+pub fn check_version() {
+    // All this to look up `HelixRuntime::VERSION`!
```

Comment:
Instead of checking the Ruby `HelixRuntime::VERSION`, you could just make the C-extension expose an additional real symbol 😛  At the end of the day that is what we are interfacing with, so checking the symbol will probably be more reliable (and easier) than checking a Ruby value.